build.sbt

```diff
@@ -2,21 +2,44 @@ organization in ThisBuild := "com.ericsson.jenkinsci.hajp"
 
 scalaVersion in ThisBuild := "2.11.5"
 
-// default credentials needed to resolve dependencies for compilation
-// in order to publish, you require credentials that have write access.
-// These can be placed in ~/.sbt/0.13/plugins/credentials.sbt
-
-// ivy 2 has a problem with snapshot updates, it does not correctly resolve them unless and until we force maven repo to be
-// first in resolver chain like this. := puts it on top of the sequence basically.
-resolvers in ThisBuild := ("Local Maven Repository" at "file:///" + Path.userHome.absolutePath + "/.m2/repository") +: resolvers.value
```

Comment:
Do not remove that... this is useful for local snapshot related testing.. it is related to an unresolved SBT bug, basically SBT does not deal well with snapshot release updates... once  ivy2 takes it, it does not update no matter how many times you create one from maven.